packages/plugin-align/src/Align.ts

```diff
@@ -0,0 +1,86 @@
+
+export enum AlignType {
+    LEFT = 'left',
+    CENTER = 'center',
+    RIGHT = 'right',
+    JUSTIFY = 'justify',
+}
+export type AlignParams = CommandParams & {
+    type: AlignType;
+};
+
+export class Align<T extends JWPluginConfig = JWPluginConfig> extends JWPlugin<T> {
+    static dependencies = [];
+    editor: JWEditor;
+    commands = {
+        align: {
+            handler: this.align.bind(this),
+        },
+    };
+    loadables: Loadables<Keymap> = {
+        shortcuts: [
+            {
+                pattern: 'Ctrl+Shift+L',
+                commandId: 'align',
+                commandArgs: { type: AlignType.LEFT },
+            },
+            {
+                pattern: 'Ctrl+Shift+E',
+                commandId: 'align',
+                commandArgs: { type: AlignType.CENTER },
+            },
+            {
+                pattern: 'Ctrl+Shift+R',
+                commandId: 'align',
+                commandArgs: { type: AlignType.RIGHT },
+            },
+            {
+                pattern: 'Ctrl+Shift+J',
+                commandId: 'align',
+                commandArgs: { type: AlignType.JUSTIFY },
+            },
+        ]
+    }
+
+    //--------------------------------------------------------------------------
+    // Public
+    //--------------------------------------------------------------------------
+
+    /**
+     * Align text.
+     */
+    align(params: AlignParams): void {
+        const nodes = params.context.range.targetedNodes(node => !node.atomic);
+        let textAlign = 'text-align: ' + params.type.toLowerCase();
+        for (const node of nodes) {
+            if (typeof node.attributes.style === 'string') {
+                if (node.attributes.style.includes('text-align')) {
+                    // Replace the existing alignment.
+                    const styles = node.attributes.style.split(/;\s*/);
+                    const alignIndex = styles.findIndex(style => style.startsWith('text-align'));
+                    styles[alignIndex] = textAlign;
+                    node.attributes.style = styles.join('; ');
+                } else {
+                    // Add an alignment to the existing styles.
+                    node.attributes.style = [textAlign, node.attributes.style].join('; ');
+                }
+            } else {
+                // Create a new style attribute, with an alignment.
+                node.attributes.style = textAlign;
+            }
+            // Remove unnecessary space from the style attribute.
+            node.attributes.style = node.attributes.style.replace(/^\s*|\s*$/g, '');
+            if (!node.attributes.style.length) {
+                // If the style attribute is empty, remove it.
+                delete node.attributes.style;
+            } else if (!node.attributes.style.endsWith(';')) {
+                // Ensure the style attribute ends with a semicolon.
+                node.attributes.style += ';';
```

Comment:
Just curious, why ?

Reply:
Standards... I'm used to seeing styles ending with ';' in html and it bothers me if it doesn't. Is that wrong doctor?